CMakeLists.txt

```diff
@@ -10,12 +10,6 @@ if (VTK_USE_X)
   list(APPEND classes vtkXLookingGlassRenderWindow)
 elseif (WIN32)
   list(APPEND classes vtkWin32LookingGlassRenderWindow)
-  if (NOT VTK_USE_MICROSOFT_MEDIA_FOUNDATION OR NOT VTK_USE_VIDEO_FOR_WINDOWS)
-    message(FATAL_ERROR
```

Comment:
For reference, there is currently described as a hard requirements:
* https://github.com/Kitware/LookingGlassVTKModule/commit/2cf690d13017cbd800c631f2e9b3672c7da56f98
* https://github.com/Kitware/LookingGlassVTKModule/blob/ce561eabf3eb2207c828c596a50fb3f9cb2028f7/vtkLookingGlassInterface.cxx#L80-L85
* https://github.com/Kitware/VTK/blob/master/IO/Movie/vtkMP4Writer.cxx

In `vtkMP4Writer`, we can see the dependence on these headers:

```
#include <mfapi.h>
#include <mferror.h>
#include <mfreadwrite.h>
```

Which implied a dependency on [Microsoft Media Foundation](https://docs.microsoft.com/en-us/windows/win32/api/_mf/)
* https://docs.microsoft.com/en-us/windows/win32/api/mfreadwrite/
* https://docs.microsoft.com/en-us/windows/win32/api/mfapi/

It may be sensible to introduce a build option to disable the export to video and/or revisit the interface to enable export leveraging the fact `vtkGenericMovieWriter` derives from `vtkImageAlgorithm`

Reply:
In master, it is a hard requirement, but I removed that in this PR. See [here](https://github.com/Kitware/LookingGlassVTKModule/pull/34/files#diff-0b8570a4184c4ed295487030edc20699be05c232019f13b47b1e50d3355a328fR75).

Reply:
FYI: apparently the Windows python wheels are currently not built with MP4 support, so we really do need to use Ogg Theora for them.

But I agree that being able to provide a `vtkGenericMovieWriter` would be more flexible so that the user has the freedom to use whatever movie writer they would like.